src/Middleware/RedirectToLogin.php

```diff
@@ -29,19 +31,36 @@
  */
 class RedirectToLogin implements MiddlewareInterface
 {
+    private State $conf;
     private Horde_Registry $registry;
     private ResponseFactoryInterface $responseFactory;
-    public function __construct(Horde_Registry $registry, ResponseFactoryInterface $responseFactory)
+    public function __construct(Horde_Registry $registry, ResponseFactoryInterface $responseFactory, State $conf)
     {
         $this->registry = $registry;
         $this->responseFactory = $responseFactory;
+        $this->conf = $conf;
     }
     public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
         if ($request->getAttribute('HORDE_AUTHENTICATED_USER')) {
             return $handler->handle($request);
         }
-        $redirect = (string)Horde::Url($this->registry->getInitialPage('horde'), true);
+        $url = (string)$request->getUri();
+        $url = Horde::signUrl($url);
+
+        // set baseurl: check if alternative login is set (currently this part is working)
+        $configArray = $this->conf->toArray();
+        $alternateLogin = $configArray['auth']['alternate_login'];
+        if(!empty($alternateLogin)){
```

Comment:
Spaces before and after the condition. Are you using a autoformatter?

Reply:
Ok done.

yes Im using php-cs-fixer, it does not seems to add these spacings per default... maybe there are some rules one add not sure